protocol/contracts/trucoV1/Structs.sol

```diff
@@ -10,6 +10,7 @@ library CardsStructs {
 
     enum Action {
         PlayCard,
+        EnvidoCount,
```

Comment:
The problem with this is that i thinked about this in a model of "action  -> response", beaing  `EnvidoCount` clearly in the second category.

For case you describe it i would implemented as:
```
Turn: PLayer 1
Challenge: Envido
Response:  Accept

Turn: Player2
Challenge: RealEnvido
```

So, my arquictecture suggestion was a "multi step action parsing" approach, so when something that in real life game would be "Envido! / RealEnvido!" in our model that challenge should be parsed as a list of allowed challenges and responses, switching turns